loader/pom.xml

```diff
@@ -7,23 +7,18 @@
 		</repository>
```

Comment:
The pom file has conflicts with the dev branch.

Reply:
I will fix.

Reply:
I have fixed it now. Can you resolve the conversation?

Reply:
It still shows conflicts.

Reply:
I think the only way of solving the conflict is merging (at least this is what github suggests). So take a look to the commit, check that the pom.xml is correct and resolve this.